nanome/_internal/_structure/_io/_pdb/structure.py

```diff
@@ -57,6 +58,18 @@ def structure_molecule(atoms, compnds):
                 all_chains[chain_id]._add_residue(residue)
             all_residues[residue_id]._add_atom(atom)
             all_atoms[atom_id] = atom
+        conect_lookup[atom._serial] = atom
+    for conect_list in conects.values():
+        serials = conect_list[-1].atoms_serial_numbers
+        atom1, atom2 = [conect_lookup[i] for i in list(serials)]
+        bond = _Bond()
+        bond._atom1 = atom1
+        bond._atom2 = atom2
+        bond._kind = list(_Bond.Kind)[len(conect_list)]
```

Comment:
I don't think this is right... according to [this](https://www.wwpdb.org/documentation/file-format-content/format33/sect10.html):
> The connectivity list given here is redundant in that each bond indicated is given twice, once with each of the two atoms involved specified in columns 7 - 11

Wouldn't that mean all of the bonds are interpreted as double bonds?